themes/default/content/docs/using-pulumi/pulumi-packages/debugging-provider-packages.md

```diff
@@ -67,3 +67,49 @@ Once Pulumi runs or tests are initiated with the `PULUMI_DEBUG_PROVIDERS` enviro
 {{< notes type="warning" >}}
 **Terminating the Provider Process**: Be cautious when terminating the provider process as the Pulumi state can get out of sync with the actual cloud resources. When in doubt, `pulumi refresh` will address this.
 {{< /notes >}}
+
+## Debug tfgen
```

Comment:
It feels like this should go in https://github.com/pulumi/pulumi-terraform-bridge/tree/master/docs instead of `debugging-provider-packages.md`, since it is highly bridge specific.

Reply:
I agree that it is bridge specific, but I think there is a greater benefit to having debugging information in a single place.

If you think it's a blocker I'll go ahead and move it there.